_redirects

```diff
@@ -3,3 +3,4 @@
 /docs/components/kustomize/controller       /docs/components/kustomize      301!
 /docs/components/notification/controller    /docs/components/notification   301!
 /docs/components/source/controller          /docs/components/source         301!
+/docs/guides/installation                   /docs/installation              301!
```

Comment:
I'm quite certain this already worked in the past, but now it doesn't.

Apparently the problem is that we don't deploy `_redirects` into `/` (cf https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file)

The easiest way to do this now is:

```cli
git mv _redirects static/
```

If you move it as part of this PR, I'll make sure to document it as part of #237.

Reply:
Done

Reply:
Thanks a lot - it works now! https://609bc5434a75050007e1eb17--fluxcd.netlify.app/docs/guides/installation

Reply:
I put it into #256